src/views/blocks/tickets/content-description.php

```diff
@@ -16,13 +16,26 @@
  */
 
 $ticket = $this->get( 'ticket' );
+$modal  = $this->get( 'is_modal' );
+$id = 'tribe__details__content' . ( true === $modal ) ?: '__modal';
+$id .= '--' . $ticket->ID;
 
-if ( ! $ticket->show_description() ) {
+if ( ! $ticket->show_description() || empty( $ticket->description ) ) {
 	return false;
 }
 ?>
-<div
-	class="tribe-block__tickets__item__content__description"
->
+<?php if ( true === $modal && $ticket->show_description() && ! empty( $ticket->description ) ) : ?>
+		<div class="tribe-block__tickets__item__details__summary">
+			<div
+				class="tribe-common-b3 tribe-block__tickets__item__details__summary--more"
+				aria-controls="<?php echo esc_attr( $id ); ?>"
+				tabindex="0"><span class="screen-reader-text"><?php esc_html_e('Open the ticket description.', 'event-tickets'); ?></span><?php echo esc_html_x('More', 'Opens the ticket description', 'event-tickets'); ?></div>
```

Comment:
the indentation is weird here, also break into multiple lines. same applies to the one below.